lib/ember-test-helpers/build-registry.js

```diff
@@ -47,6 +47,14 @@ export default function(resolver) {
     registry = new Ember.Registry({
       fallback: fallbackRegistry
     });
+
+    // these properties are set on the fallback registry by `buildRegistry`
+    // and on the primary registry within the ApplicationInstance constructor
+    // but we need to manually recreate them since ApplicationInstance's are not
+    // exposed externally
+    registry.normalizeFullName = fallbackRegistry.normalizeFullName;
+    registry.makeToString = fallbackRegistry.makeToString;
```

Comment:
Updated to add `.describe`, please let me know what you think of a `.wrap` method or somesuch to make this cleaner from `ApplicationInstance` / `ember-test-helpers` perspective.